Update troubleshooting.md (#15259)
DataDog/documentation: content/en/real_user_monitoring/session_replay/troubleshooting.md

Session Replay does not support the following HTML elements: `iframe`, `video`, `audio`, or `canvas`. Session Replay does not support Web Components.

> Session Replay also *requires* a HTTPS connection, if you aren't using a secure connection the resources will time out and you will be unable to see images and some other page elements.

Reviewer: ```suggestion
Session Replay requires you to use an HTTPS connection. If you aren't using a secure connection, the resources time out and you can't see images and some page elements.
```
